packages/menu/src/Menu/Menu.stories.args.js

```diff
@@ -0,0 +1,84 @@
+import { colors } from '../../../core/storybook/args'
+
+export const defaultArgs = {
+  isOpen: true,
+  buttonText: 'Menu',
+}
+
+export const argTypes = {
+  id: {
+    name: 'id',
+    type: { name: 'string', required: false },
+    description: 'DOM id',
+    table: {
+      defaultValue: { summary: undefined },
+    },
+  },
+  isOpen: {
+    name: 'isOpen',
+    type: { name: 'boolean', required: false },
+    description: 'Is the menu open?',
+    table: {
+      defaultValue: { summary: true },
+    },
+    control: {
+      type: 'boolean',
+    },
+  },
+
+  buttonText: {
+    name: 'buttonText',
+    type: { name: 'string', required: false },
+    description: 'text on Button trigger',
+    table: {
+      defaultValue: { summary: undefined },
+    },
+  },
```

Comment:
n-b: maybe at some point we can make helper functions to shorten up some of the args files. It looks like a lot of props are common between stories.

Reply:
oh definitely. this PR really just sets the stage